src/layers/domain/core/cpm_product/v1.py

```diff
@@ -31,12 +34,26 @@ class CpmProductKeyAddedEvent(Event):
     product_team_id: UUID
     name: str
     ods_code: str
+    status: Status
     created_on: str
     updated_on: str = None
```

Comment:
Will this still default to None for updated on? Or will it always get updated when a key is added?

Reply:
yes this will default to `None`